composer.json

```diff
@@ -57,6 +57,13 @@
             ]
         }
     },
+    "scripts": {
+        "test": "./vendor/bin/phpunit",
+        "test:group": "./vendor/bin/phpunit --group test",
```

Comment:
We don't have `@group test` in the codebase. What's the purpose of this?

I use `./vendor/bin/phpunit --filter 'ModelTest::testNewModel'` to target a specific test (or PHPStorm PHPUnit integration of course!).

Reply:
Yes, that's true. `--group` is another option to run the tests. we can select the tests that we want to execute by adding a docblock.
```php
/** @group test */
public function testWithCountHasMany(): void {}
```
The advantage is, that you can select one or more tests from several classes.
I often write multiple tests for a PR. I add this docblock to my tests to ensure that my changes don't break the previous tests. I prefer this method because adding a docblock to the code is easier (for me at least).

Reply:
Please revert.